server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -181,7 +185,11 @@ export class MetadataExtractionProcessor {
       }
 
       await this.exifRepository.upsert(newExif, { conflictPaths: ['assetId'] });
-      asset = await this.assetCore.save({ id: asset.id, fileCreatedAt: fileCreatedAt?.toISOString() });
+      asset = await this.assetCore.save({
+        id: asset.id,
+        fileCreatedAt: fileCreatedAt?.toISOString(),
+        ...(!asset.sidecarPath && sidecarPath && { sidecarPath }),
```

Comment:
I think it is OK to just set it every time. Eg.

```typescript
.save({ 
  ...
  sidecarPath
  ...
})
```

Reply:
This was 'resolved' as the saving of the asset was moved into the SYNC_DISCOVER job, so it always saves the `sidecarPath` property.